client/src/pages/Home.js

```diff
@@ -1,30 +1,78 @@
 import MainTable from "../Component/MainTable";
 import SMSModal from "../Component/SMSModal";
 import "./Home.css";
-import { useState } from "react";
-import UploadFile from "./UploadFile";
+import React, { useState, useEffect } from "react";
+import Alert from "react-bootstrap/Alert";
+
+
 
 export function Home() {
-	const [checkedCheckboxes, setCheckedCheckboxes] = useState([]);
+	const [selectedClients, setSelectedClients] = useState([]);
+	const [clientData, setClientData] = useState([]);
+	const [error, setError] = useState(null);
+	const [isLoading, setIsLoading] = useState(false);
+	const [success, setSuccess] = useState(false);
+	const [danger, setDanger] = useState(false);
```

Comment:
```suggestion
	const [isFailure, setIsFailure] = useState(false);
```
`danger` reads a bit strangely to me as another developer as there's no danger here. What do you think about changing the name to something like `isFailure` instead?

Reply:
haha, it is a bit over the top!